### DISCUSSION TITLE
Honored Reputation Requirement for WSG and AB PvP Epic Equipment

### DISCUSSION THREAD

**Discussion by plasma-93** [chromiecraft/chromiecraft#3482] [Suggestions]
The reputation grind for AV and WSG are incredibly tedious, and the time requirement is definitely leaning on the 'absurd' side of things. 

I personally believe that the reputation grind for WSG and AB are bad for the server, and doesn't really vibe well with a predominately casual player base. Reasoning:

- The gear is already being outpaced with some BWL gear so it's not like you'd leapfrog content by obtaining it. There are a few cases for hybrid classes that didn't have supporting tier sets to go for this gear, but most classes have better T2 pieces.

- The reputation requirement was removed in Wrath for all PvP gear, I can't actually come up with any valid reasons why it was reimplemented either, except maybe a misguided attempt to increase players engagement with PvP, but around 60-70% of people I asked (anecdotal and biased, tbf) said that the reason they haven't farmed is because the time investment is ridiculous. 

- Time investment, in general, I don't like changes like requiring ony / bwl attunement personally, but they haven't had much of an adverse effect, and there's been some positives, like giving us something to do with guildies. The change to include reputation grinds in Wrath WSG, AV, and AB added a ridiculous amount of time investment for rewards from these dungeons that should not be there. AV was lowered to Honored to fix this, yet WSG and AB have not. Currently it takes between 2 and 5 hours to get items from AV, and around 360-700hours to get rewards from WSG and AB. If you're really bad, and unlucky, this time is maximally around 29days of played time. This is made worse by PvP queues not always popping, or being really slow, such as 2v2,WSG and 4v4AB.

- There is already an honor cost tied to the items as their cost. Why not increase this cost, 50k per reputation item, and 75k per piece of Rank12-14 items. Rank12-14 gear is already being contested by the really nice BWL gear, not sure why the armor pieces haven't been released yet, bit of a tangent.

-  Making the items attainable will increase the amount of people willing to try PvP, and may stick with it, which is better for the PvP scenes health.

Also here's a fun little spreadsheet Aponi made when I complained about this problem in the pvp channel.
Time to get Exalted for WSG, it is.. kind of insane.
![image](https://user-images.githubusercontent.com/38546519/167312782-ca69c2f7-2927-4026-af69-2cd0f235efd7.png)

I'm not asking to get the items for nothing, but it's insane to ask for half a month of game-time, imo. I spent my entire weekend, pushing my responsibilities aside just to try and get HONORED for AB, I think I'll barely do it on a CTA, that's a HUGE ask for exalted. They removed stuff like this in wrath for a reason, this feels particularly counterintuitive to pattern-recognizable changes to Wrath item acquisition, 

This all juxtaposes really weirdly with AV, which takes less than 2 hours on a good day for Honored... Why such a drastic difference? Especially since Honored with AV was so well received?

_My suggestion_ is to reduce the rep required to Honored, or to remove the rep required entirely since it was removed in Wrath, for the better. I can't personally think of good reasoning for having the reputation requirement, but if it must be there, Honored is already a lot of PvP for most people.

**Comment by KaneoHunter** [comment]
Please god PLEASE

**Comment by xmaxvc24** [comment]
Pls , make it real :)

**Comment by Vitusin** [comment]
Like if you want to make it ,,blizzlike" just remove the rep. need or put it to honored. We need more players in pvp to try and enjoy. Please, listen to what the players are telling you. Make quality life decision for players.

**Comment by Ithiloneth** [comment]
I support this!

I want to add that it is important to understand that it is not just the time required to actually play, but that the BGs aren't active.
I pushed as hard as I could during AB CtA this weekend, I did not get far. When is the next AB CtA? When will I be done?
By the time the gear has become obsolete I bet.

Sour grapes? At that height yes; **very** sour.

**Reply by Ithiloneth** [reply]
As an aside - AV rep grind was plain easy, and done by me simply by playing PvP during a single CtA. It was fun though - good two days of PvP and a reward at the end; just right!

**Comment by minthes** [comment]
It would be cool even if they reduce it to revered instead of exalted. Then when AQ comes out remove the rep requirement entirely to allow new players to jump in the game more easily.

**Reply by plasma-93** [reply]
My problem with that, is if you've grinded to Honored you know it's already a pretty damn hefty grind. Grinding Cenarion Circle rep to exalted was faster than AB Honored.

**Comment by Never6** [comment]
Making all marks 250 per turn in and not just the AV ones would be a great place to start.

**Comment by Annamaria-CC** [comment]
We have implemented a reduced reputation requirement on the server. We will continuously look if it needs further adjustments. Thanks for the suggestion. Locking this thread.
